docs/source/model/dependencies.rst

```diff
@@ -438,21 +445,28 @@ To do so, use the **pip-requirements-override** option to specify pip dependenci
         import mlflow
 
         mlflow.models.predict(
-            model_uri="runs:/<run_id>/model",
+            model_uri="runs:/<run_id>/<model_path>",
             input_data=<input_data>,
             pip_requirements_override=["opencv-python==4.8.0"],
         )
 
     .. code-tab:: bash
 
         mlflow models predict \
-            -m runs:/<run_id>/model \
+            -m runs:/<run_id>/<model_path> \
             -i <input_path> \
```

Comment:
```suggestion
            -I <input_path> \
```
This was my mistake, but the docstring change below make me realize (the arg is case sensitive).